wiki/Tournaments/IGTS/2018/en.md

```diff
@@ -41,30 +41,148 @@ The Intermediate Global Taiko Showdown 2018 is run by various osu!taiko communit
 | Map Selectors | ![][flag_CA] [FlamingRok](/users/2015353), ![][flag_PT] [babysnakes](/users/4669728), ![][flag_ID] [Poii](/users/7924938) |
 | Designer | ![][flag_US] [Swovine](/users/6666316) |
 | Statisticians | ![][flag_FR] [Kasumii-sama](/users/6177263),  ![][flag_FR] [TLQ_Yoshii](/users/7157133) |
-| Referees | ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [TLQ_Yoshii](/users/7157133), ![][flag_AU] [cactusthegreat](/users/5592409), ![][flag_US] [ITotallyGetThat](/users/8770622) |
-| Streamer | ![][flag_DE] [Heamburger](/users/4705120) |
-| Casters | ![][flag_MY] [Bedwyr Aorta](/users/10875855), ![][flag_EC] [Gamelan4](/users/9856910) |
+| Referees | ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [TLQ_Yoshii](/users/7157133), ![][flag_TR] [oralekin](/users/7631823), ![][flag_AU] [cactusthegreat](/users/5592409), ![][flag_US] [ITotallyGetThat](/users/8770622), ![][flag_JP] [Briesmas](/users/2865172), ![][flag_JP] [Noko_BSF](/users/3811831), ![][flag_RU] [-\_-Nico-\_-](/users/9689534) |
+| Streamers | ![][flag_DE] [Heamburger](/users/4705120), ![][flag_US] [Swovine](/users/6666316), ![][flag_US] [ITotallyGetThat](/users/8770622), ![][flag_MY] [Bedwyr Aorta](/users/10875855) |
+| Casters | ![][flag_DE] [Heamburger](/users/4705120), ![][flag_EC] [Gamelan4](/users/9856910), ![][flag_AR] [BossPlays_02](/users/7341471), ![][flag_MY] [Bedwyr Aorta](/users/10875855), ![][flag_US] [CaptainEChan](/users/9536977), ![][flag_US] [Edgar_Figaro](/users/6508754) |
+| Wiki | ![][flag_ID] [fajar13k](/users/7100002) |
 
 ## Links
 
 - [Discussion Thread](/community/forums/topics/767841)
 - [IGTS Discord](https://discord.gg/qavX3Ga)
-- [Livestream](https://www.twitch.tv/beginnerstaikotourney)
+- [Livestream](https://www.twitch.tv/igtsosu)
 - [Player/Team Registration Form](https://docs.google.com/forms/d/e/1FAIpQLScmGzj3NMAOimVUzSdMRDa8COV5MOxcANOljLuxQjrJ9wLIvg/viewform)
 - [Staff Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSe39Rk4JyUQ_GGBJHdYxfA7kZmJQqboK3p1LV_eTYZf-8Td_Q/viewform)
 - [Challonge Brackets](https://challonge.com/u85xl34m)
 
 -----------------------
 
+## Match Schedule: Group Stage
+
+### Saturday, 04 August 2018
+
+| Match ID | Team A |  |  | Team B | Match Time (UTC) | Group |
+| :-: | --: | :-: | :-: | :-- | :-: | :-: | :-: | :-: |
```

Comment:
Remove the last 2 alignments, those are breaking the table (unless you wanted to add the local times, if thats the case those are missing)
Repeat on every table in schedules.